src/NServiceBus.NHibernate/TimeoutPersisters/TimeoutEntity.cs

```diff
@@ -29,18 +29,26 @@ class TimeoutEntity
         public virtual byte[] State { get; set; }
 
         /// <summary>
+        /// Timeout endpoint name.
```

Comment:
This alone will not give us a clustered index on (Endpoint, Time) on SQL Server. It is still needed for proper order column on other databases (read: Oracle).